src/main/java/by/exadel/internship/exception_handing/IncorrectData.java

```diff
@@ -0,0 +1,12 @@
+package by.exadel.internship.exception_handing;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class IncorrectData {
+    private String info;
```

Comment:
Add code field. It will look approximatly like "user.invalid". We think UI will need "code" to show errors.

Reply:
done